program-nrf

```diff
@@ -0,0 +1,4 @@
+make TOCK_PLATFORM=nrf_pca10001 -C apps/c_gpio 
```

Comment:
This script is great, but should be a `make` target in `Makefile.nrf_pca10001`. E.g.:

```Makefile
$(TOCK_APP_BUILD_DIR)/kernel_and_app.hex: $(TOCK_APP_BUILD_DIR)/kernel_and_app.elf
    @tput bold ; echo "Flattening $< to $@" ; tput sgr0
    $(OBJCOPY) -O ihex $< $@

.PHONE: program-mbed
program-mbed: $(TOCK_APP_BUILD_DIR)/kernel_and_app.hex: $(TOCK_APP_BUILD_DIR)/kernel_and_app.elf
    cp $^ $(MOUNT_DIR)/
```